README.md

```diff
@@ -220,6 +220,15 @@ You can also specify it during instantiation.
 const agenda = new Agenda({mongo: mongoClientInstance});
 ```
 
+#### Using a mongoose connection
+To share the connection pool with an already existing mongoose connecting, pass mongoose.connection to agenda.
```

Comment:
Is this meant to be "mongoose connection"?

Reply:
yes! thanks